### PR TITLE
Fixed presets

### PR DESCRIPTION
**Pull request by WolframRavenwolf**
Updates for recently added presets that improve their compatibility and fix some issues I've noticed while testing them extensively:

- **Update KoboldAI Deterministic.settings:** After extensive testing, I've adjusted repetition penalty slightly to be the same as simple-proxy-for-tavern's default preset and ooba's LLaMA-Precise settings preset. This fixed some models talking/acting as User.

- **Update TextGen Deterministic.settings:** This wasn't actually deterministic because despite `do_sample: false`, temperature was still taking effect, causing non-deterministic output. I fixed this and also adjusted repetition penalty to be like KoboldAI's Deterministic preset.

- **Set TextGen temperature slider min value to 0.0:** When updating the TextGen preset, I noticed the slider not going to the true minimum, so I fixed that as well. Same as [#825](https://github.com/SillyTavern/SillyTavern/issues/825) for KoboldAI.

- **Update Roleplay instruct mode preset: Include Names:** After extensive testing, I've enabled "Include Names" to fix some models talking/acting as User.

### CODE REVIEW

File: public/instruct/Roleplay.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "input_sequence": "### Instruction:",
```
Comment (Cohee1207) [inline public/instruct/Roleplay.json:2]: You may want to replace "include names" option with adding `\n####{{user}}\n` for better parity with simple proxy

Comment (WolframRavenwolf) [reply public/instruct/Roleplay.json:2]: I actually had that in the original version when creating the Roleplay preset - I still have it locally as an almost exact copy of what simple-proxy does:

```
{
    "input_sequence": "### Instruction:\n#### {{user}}:",
    "macro": true,
    "name": "simple-proxy-for-tavern",
    "names": false,
    "output_sequence": "### Response:\n#### {{char}} (2 paragraphs, engaging, natural, authentic, descriptive, creative):",
    "separator_sequence": "",
    "stop_sequence": "",
    "system_prompt": "## {{char}}\n- You're \"{{char}}\" in this never-ending roleplay with \"{{user}}\".\n### Input:",
    "system_sequence": "",
    "wrap": true
}
```

That's as close to the original simple-proxy as I could get. However, in my extensive testing, I didn't see a real benefit for the additional tokens it cost, so I removed everything I considered unnecessary and turned it into the simplified Roleplay preset we have now.

If you want, I can add that to the PR?

Still, after testing the Roleplay preset with several dozens of models over dozens of sessions for dozens of hours, I'd recommend that over the simple-proxy-for-tavern preset which does the same but uses up a whole bunch of additional tokens. The actual proxy at least has the advantage that " (2 paragraphs, engaging, natural, authentic, descriptive, creative)" would only be added to the final output sequence, not to all, but we don't have that possibility in SillyTavern yet.

Comment (Cohee1207) [reply public/instruct/Roleplay.json:2]: Okay, I'll make sure to put the ability to define the last output sequence to my backlog. I trust your judgement here, so let's merge as is for now

**Comment by Cohee1207** [comment]
@StefanDanielSchwarz 
Last Output Prompt and extended story string formatting were added to staging. 
You should now be able to replace proxy formatting as close as possible.

**Comment by Cohee1207** [comment]
If you wish to dig into story string templates syntax - take a look at /public/context/ JSON files

**Comment by WolframRavenwolf** [comment]
Oh, wow, that was fast. I'll check it out as soon as I can.

When do you plan to release the next stable version with that feature?

**Comment by Cohee1207** [comment]
Possibly not very soon. We're far from stable now, especially in the non-free models' field (see #951 ).

**Comment by WolframRavenwolf** [comment]
OK, good to know. I've taken a first look and this is very helpful stuff. Noted down half a dozen issues so far and will look into fixing them over the weekend or reporting them if I can't do it myself. But great start already, thanks for adding this!

**Pull request merged**